### PR TITLE
List candidates

### PR DESCRIPTION
**Pull request by JamBai2812**
Got working with some dummy data, added links to pages that don't exist yet.

Need to add real data when api endpoint is finished.

### CODE REVIEW

File: components/CandidateList/CandidateList.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import scss from './CandidateList.module.scss';
+import {getCandidates} from "../../api/candidatesApiClient";
+import {CandidateLink} from "../CandidateLink/CandidateLink";
+
+
+export function CandidateList(): JSX.Element {
+
+    //TODO: use getCandidates function to get actual data
+    
+    const candidates: string[] = ["Michael Walker", "Stephen Jackson"];
```
Comment (MichaelWalker) [inline components/CandidateList/CandidateList.tsx:11]: same here - if we create a little model for a Student with a Name and an Id and use that instead then we'll probably have to make far fewer changes later on!

File: components/CandidateList/CandidateList.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    
+    return (
+        <ul className={scss.list}>
+            {
+                candidates.map(name => <CandidateLink key={candidates.indexOf(name)} name={name}/>)
```
Comment (MichaelWalker) [inline components/CandidateList/CandidateList.tsx:16]: js gives us a neater syntax for this! :) 
(or once you have the Id you can just use that)

```ts
candidates.map((name, index) => <CandidateLink key={index} name={name}/>)
```

File: components/CandidateLink/CandidateLink.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+
+interface CandidateLinkProps {
+    name: string;
+}
+
+
+export function CandidateLink(props: CandidateLinkProps): JSX.Element {
+
+    const ref: string = "/candidates/" + props.name;
```
[the remark this answers is not in the document]
Comment (JamBai2812) [reply components/CandidateLink/CandidateLink.tsx]: changed!

File: components/CandidateList/CandidateList.tsx (hunk 1 above)
Comment (JamBai2812) [reply components/CandidateList/CandidateList.tsx]: changed!

File: components/CandidateList/CandidateList.tsx (hunk 2 above)
Comment (JamBai2812) [reply components/CandidateList/CandidateList.tsx]: used index for this

File: pageStyles/candidates.module.scss [hunk 4]
```diff
@@ -0,0 +1,2 @@
+﻿@import 'constants';
+
```
Comment (MichaelWalker) [inline pageStyles/candidates.module.scss:2]: Looks like we can just kill this now?

Comment (JamBai2812) [reply pageStyles/candidates.module.scss]: yep

**Pull request merged**